fhir-server/src/test/java/au/csiro/pathling/aggregate/AggregateQueryTest.java

```diff
@@ -385,6 +385,21 @@ void queryWithNestedAggregation() {
         response);
   }
 
+  @Test
```

Comment:
This test is OK but the one above `queryWithNestedAggregation()` does not seem to be testing a nested aggregation of the same type so I am not sure if nested aggregation is covered for the case with grouping present.

Reply:
See eee8f3b.